website/search/elastic_search.py

```diff
@@ -85,6 +85,9 @@ def wrapped(*args, **kwargs):
                 if 'ParseException' in e.error:
                     raise exceptions.MalformedQueryError(e.error)
                 raise exceptions.SearchException(e.error)
+            except TransportError as e:
+                # Catch and wrap generic ES 500 codes. TODO: Improve fix for https://openscience.atlassian.net/browse/OSF-4538
```

Comment:
As I read the elasticsearch client ([one](https://github.com/elastic/elasticsearch-py/blob/1.3.0/elasticsearch/connection/http_requests.py#L69-L72), [two](https://github.com/elastic/elasticsearch-py/blob/1.3.0/elasticsearch/connection/base.py#L91-L102)), `TransportError` or a [subclass](https://github.com/elastic/elasticsearch-py/blob/1.3.0/elasticsearch/exceptions.py#L77-L103) is used for any response code greater than or equal to 300.